ui/user/src/lib/components/mcp/McpServerInfo.svelte

```diff
@@ -261,41 +242,14 @@
 	<div
 		class="dark:bg-surface1 dark:border-surface3 flex h-fit flex-col gap-4 rounded-lg border border-transparent bg-white p-4 shadow-sm md:w-1/2 lg:w-8/12"
 	>
-		{#if editable}
-			{#if editDescription}
-				<MarkdownTextEditor
-					bind:value={description}
-					initialFocus
-					onUpdate={handleDescriptionUpdate}
-					onCancel={() => (editDescription = false)}
-				/>
-			{:else if description}
-				<div class="group relative w-full">
-					<div class="milkdown-content">
-						{@html toHTMLFromMarkdownWithNewTabLinks(description)}
-					</div>
-					<button
-						class="icon-button absolute top-0 right-0 z-10 min-h-8 opacity-0 transition-all group-hover:opacity-100"
-						onclick={() => (editDescription = true)}
-					>
-						<Pencil class="size-5 text-gray-400 dark:text-gray-600" />
-					</button>
-				</div>
-			{:else}
-				<button
-					class="group relative flex min-h-8 w-full justify-between gap-2 pt-0 text-left"
-					onclick={() => (editDescription = true)}
-				>
-					<span class="text-md text-gray-400 dark:text-gray-600">Add description here...</span>
-					<div class="icon-button opacity-0 group-hover:opacity-100">
-						<Pencil class="size-5 text-gray-400 dark:text-gray-600" />
-					</div>
-				</button>
-			{/if}
-		{:else if description}
+		{#if description}
 			<div class="milkdown-content">
 				{@html toHTMLFromMarkdownWithNewTabLinks(description)}
 			</div>
+		{:else}
+			<p class="text-md text-center font-light text-gray-500 italic">
+				{descriptionPlaceholder}
+			</p>
```

Comment:
Note: A decent side effect of this is that we now get the `No description available` instead of an empty box in every other context where `McpServerInfo` is being used; e.g. chat and `/mcp-servers`

<img width="1712" height="999" alt="Image" src="https://github.com/user-attachments/assets/0577b2c7-e6fe-4180-97ea-1351bae5818e" />